README.md

```diff
@@ -1,24 +1,67 @@
-# README
+# geo.lclark.edu
 
```

Comment:
add `<img src="https://travis-ci.org/WatzekDigitalInitiatives/geo.lclark.edu.svg?branch=master" alt="travis status">` for travis status

Reply:
@thatbudakguy i added markdown version of the same